tests/zlib-simple/expected/data/out/files/test_zlib/test_zlib

```diff
@@ -0,0 +1 @@
+[{"GeneralInfo":[8,1,1562108092.7915649,1562108092.878231,"iPhone","iPhone7,1","12.3.1","7E2A1F36-AC9D-4F26-9C42-4A0898A6D73D"]},{"GeneralData":["iPhone7,1","iPhone","iOS","12.3.1","iPhone","7E2A1F36-AC9D-4F26-9C42-4A0898A6D73D",false,false,true,false,true,63989469184,49519423488,1016070144,"vi","LA","LAK","LAK","vi_LA","vi-VN",null,null,true,"Asia\/Vientiane","GMT+7",true,"LaoTel","la","457","01","CTRadioAccessTechnologyHSDPA"]}]
```

Comment:
urcite udelej nejaky testovaci data (nejlepsi by bylo vymazat tohle z git historie - buhvi co tohle je!)

Reply:
Ten comment bude asi potřeba smazat, jinak to ostane součást toho PR